st2api/st2api/controllers/v1/actions.py

```diff
@@ -235,9 +236,51 @@ def delete(self, ref_or_id, requester_user):
             ref_or_id,
             action_db,
         )
+        
+        BASE_PATH = cfg.CONF.system.base_path
+        pack_name = action_db["pack"]
+        entry_point = action_db["entry_point"]
+        metadate_file = action_db["metadata_file"]
+        action_entrypoint_file_path = os.path.join(
+            BASE_PATH, "packs", pack_name, "actions", entry_point
+        )
+        action_metadata_file_path = os.path.join(
+            BASE_PATH, "packs", pack_name, metadate_file
+        )
```

Comment:
The pack base path can be overridden (ref: https://docs.stackstorm.com/packs.html#under-the-hood-pack-basics). Therefore, it is better to use get_pack_base_path @ https://github.com/StackStorm/st2/blob/master/st2common/st2common/content/utils.py#L135 to determine the pack file path.